### PR TITLE
Add a tolerance to the range check for the probability in ActivityProfileState

### PR DESCRIPTION
**Pull request by droazen**

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/utils/activityprofile/ActivityProfileState.java [hunk 1]
```diff
@@ -24,7 +28,7 @@ public double isActiveProb() {
      * @param activeProb probability between 0.0 and 1.0 that the site is active
      */
     public void setIsActiveProb( final double activeProb ) {
-        Utils.validateArg(activeProb >= 0.0 && activeProb <= 1.0, "activeProb must be >= 0.0 and <= 1.0");
+        Utils.validateArg(MathUtils.doubleWithinRangeWithTolerance(activeProb, 0.0, 1.0, PROBABILITY_TOLERANCE), "activeProb must be >= 0.0 and <= 1.0");
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/utils/activityprofile/ActivityProfileState.java]: should the message reflect that it's fuzzy?

File: src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java [hunk 2]
```diff
@@ -498,6 +498,20 @@ public static byte compareDoubles(final double a, final double b, final double e
...
+     *
+     * Does not check for NaNs or infinities.
+     *
+     * @param value value to check
+     * @param min start of allowable range
+     * @param max end of allowable range
+     * @param tolerance perform double comparisons to within this tolerance
+     * @return true if value is within the specified range (+/- tolerance), otherwise false
+     */
+    public static boolean doubleWithinRangeWithTolerance( final double value, final double min, final double max, final double tolerance ) {
```
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/utils/MathUtils.java]: seems fine

**Comment by lbergelson** [comment]
@droazen 👍

File: src/main/java/org/broadinstitute/hellbender/utils/activityprofile/ActivityProfileState.java (hunk 1 above)
Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/utils/activityprofile/ActivityProfileState.java]: Fixed

**Pull request merged**